bundled-libs/XML/RPC.php

```diff
@@ -1530,7 +1532,7 @@ function parseResponse($data = '')
                 $r = new XML_RPC_Response($v);
             }
         }
-        $r->hdrs = preg_split("@\r?\n@", $XML_RPC_xh[$parser]['ha']);
+        $r->hdrs = preg_split("@\r?\n@", array_key_exists('ha', $XML_RPC_xh[$parser]) ? $XML_RPC_xh[$parser]['ha'] : '');
```

Comment:
I would prefer:
```suggestion
        $r->hdrs = preg_split("@\r?\n@", $XML_RPC_xh[$parser]['ha'] ?? '');
```
for legibility